### PR TITLE
Documentation: Build Analysis Filtering Pipelines

### PR DESCRIPTION
**Pull request by AlitzelMendez**
Documentation for the new feature of filterin pipelines

### CODE REVIEW

File: Documentation/Projects/Build Analysis/FilteringPipelines.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Build Analysis: Filtering Pipelines
+This feature filters Azure pipelines from Build Analysis. By default, all pipelines are analyzed. If you want to analyze only specific pipelines, you need to define them in a configuration file. The filtered pipelines are exclusive to target branches, so you need to specify which pipelines are analyzed for each target branch.
+
+### Step by step getting on board.
+
+1. Create a file with the name `build-analysis-configuration.json`  on the path `eng/`  of your repository
```
Comment (missymessa) [inline Documentation/Projects/Build Analysis/FilteringPipelines.md:6]: Are we going to add this file to Arcade's `/eng` folder so folks can pull it in automatically if they don't want to add it?

**Review by missymessa — APPROVED** [review]
LGTM shipit

File: Documentation/Projects/Build Analysis/FilteringPipelines.md (hunk 1 above)
Comment (AlitzelMendez) [reply Documentation/Projects/Build Analysis/FilteringPipelines.md:6]: yes, let's do this too! just let me do more tests for this area (double check that it won't impact anyone negatively) and we can do it

Comment (AlitzelMendez) [reply Documentation/Projects/Build Analysis/FilteringPipelines.md:6]: this doesn't have any negative impact but I am having second thoughts about this

**Pull request merged**